src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        File file = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        try {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/208705801-28ee14ab-592d-485b-9207-fda578991b30.png)

Reply:
a comment about try with resources is not fixed, [here](https://stackoverflow.com/a/56151320) you can find a hint in case you are having troubles, also feel free to ask questions here or in the chat